### PR TITLE
Feat/transforms

### PR DESCRIPTION
**Pull request by nurgoni**
- Add ImageTranforms class for image transformation
- Add test for ImageTransform
- Add makefile for testing

### CODE REVIEW

File: src/lightning_classify/transforms/transforms.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+from PIL import Image
+
+import torch
+from torchvision import transforms
+
+
+class ImageTransform:
```
Comment (nunenuh) [inline src/lightning_classify/transforms/transforms.py:7]: hmmm, just make sure you can make a good reasoning later for using the transform as class

File: makefile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.PHONY: .venv
+
+run-test:
```
Comment (nunenuh) [inline makefile]: okay this is nice start for preparening makefile, but you use wrong name, it has to be in capital for the first letter, so instead makefile rename it to Makefile

File: tests/transforms/test_image_transforms.py
Comment (nunenuh) [inline tests/transforms/test_image_transforms.py:1]: this file looks good to me, have you test it?

**Review by nunenuh — CHANGES_REQUESTED** [review]
everything looks good, you just need to change makefile name to right name

File: tests/transforms/test_image_transforms.py
Comment (nurgoni) [reply tests/transforms/test_image_transforms.py:1]: image

File: makefile (hunk 2 above)
Comment (nurgoni) [reply makefile]: done

File: src/lightning_classify/transforms/transforms.py (hunk 1 above)
Comment (nurgoni) [reply src/lightning_classify/transforms/transforms.py:7]: "The reason is to simplify the number of function transformations. The test and validation transformations are the same, and the required transformation depends on whether the image is for training or not."

**Pull request merged**